config/mvg_initialize_cameras_landmarks.conf

```diff
@@ -47,6 +47,7 @@ block mvg
   essential_mat_estimator:type = vxl
 
   estimate_pnp:type = ocv
+  resection_camera:type = ocv
```

Comment:
we probably don't need this change yet.  The mvg cameras and landmarks initialization doesn't yet use this new resection API.

Reply:
OK, removed that line for now.